Cargo.toml

```diff
@@ -6,25 +6,39 @@ license = "MIT/Apache-2.0"
 description = "A wrapper over a platform's native TLS implementation"
 repository = "https://github.com/sfackler/rust-native-tls"
 readme = "README.md"
+edition = "2018"
 
 [features]
+default = [ ]
 vendored = ["openssl/vendored"]
+alpn = [ "security-framework/alpn", "security-framework-sys/OSX_10_13" ]
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
-security-framework = "0.4.1"
-security-framework-sys = "0.4.1"
-lazy_static = "1.0"
+security-framework = "=0.4.0"
+security-framework-sys = "=0.4.0"
```

Comment:
Why are these dependencies pinned to an older version?

Reply:
problem: https://github.com/kornelski/rust-security-framework/commit/8cdffd81752f68f7a476d768eacadc4e744192c8

`security-framework/Cargo.toml` not add `edition = "2018"`,
and `security-framework/src/lib.rs` not add `extern crate libc;`,
and `security-framework/src/dlsym.rs` has line `use libc;`.

this problem is fixed, but not release.